development/creds.example.env

```diff
@@ -32,15 +32,15 @@ NAUTOBOT_ARISTACV_CVP_TOKEN="changeme"
 NAUTOBOT_SSOT_INFOBLOX_PASSWORD="changeme"
 
 # ACI Credentials.  Append friendly name to the end to identify each APIC. 
-NAUTOBOT_APIC_BASE_URI_NTC=https://aci.cloud.networktocode.com
-NAUTOBOT_APIC_USERNAME_NTC=admin
-NAUTOBOT_APIC_PASSWORD_NTC=super_secret_password
-NAUTOBOT_APIC_VERIFY_NTC=False
-NAUTOBOT_APIC_SITE_NTC="NTC ACI"
-NAUTOBOT_APIC_BASE_URI_DEVNET=https://sandboxapicdc.cisco.com
-NAUTOBOT_APIC_USERNAME_DEVNET=admin
-NAUTOBOT_APIC_PASSWORD_DEVNET=super_secret_password
-NAUTOBOT_APIC_VERIFY_DEVNET=False
-NAUTOBOT_APIC_SITE_DEVNET="DevNet Sandbox"
+# NAUTOBOT_APIC_BASE_URI_NTC=https://aci.cloud.networktocode.com
```

Comment:
???

Reply:
If I leave these in, it automatically creates `Site` objects which mess up the example SSoT job.

Reply:
ok, would just check in with whomever owns aci

Reply:
isn't this a problem in a scenario when you want to use both?

Reply:
I think that we need to re-think how the ACI integration does this, its a little too invasive for my taste. I don't think me changing the development environment here should really impact anybody, so I will create an issue from this discussion.

Reply:
the integration with different controllers is gonna be implemented with the `controllers` models coming soon to Nautobot, so we could select different url endpoints easily. But, until then, I would keep as it is